schemas/README.md

```diff
@@ -133,6 +146,7 @@ Supported keys to describe fields
 - format: Field format that can be used in a Kibana index template.
 - normalize: Normalization steps that should be applied at ingestion time. Supported values:
   - array: the content of the field should be an array (even when there's only one value).
+- beta (optional): Adds a beta marker for the field to the description. The text provided in this attribute is used as content of the beta marker in the documentation.
```

Comment:
Do you agree with this approach, and is the code in line with this?

```suggestion
- beta (optional): Adds a beta marker for the field to the description. The text provided in this attribute is used as content of the beta marker in the documentation. Note that when a whole field set is marked as beta, it is not necessary nor recommended to mark all fields in the field set as beta.
```

Reply:
I agree with this approach 👍 . 

The code allows for the prescribed approach. If `beta` is set at the field set level, it will _not_ automatically apply the `beta` marker to all fields in that field set. Conversely, setting `beta` on an individual field will not automatically add the field set wide marker.